frontend/src/pages/pipelines/global/GlobalPipelineCoreDetails.scss

```diff
@@ -0,0 +1,19 @@
+// This is a hack to make the link button with an icon work in the breadcrumb.
+// Because the PF link button doesn't have an underline below the icon,
+// FIXME: We can remove this once https://github.com/patternfly/patternfly/issues/7554 is fixed.
```

Comment:
can you create a jira to track this

Reply:
Will do after this PR is merged.

Reply:
Created https://issues.redhat.com/browse/RHOAIENG-27455 to track this.